sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -591,6 +592,8 @@ def __init__(
         batch has not been completely filled up.
       retry_strategy: The strategy to use when retrying streaming inserts
         into BigQuery. Options are shown in bigquery_tools.RetryStrategy attrs.
+      additional_bq_parameters (dict, callable): A set of additional parameters
```

Comment:
Are the names of these parameters unknown, variable ? If not why not add separate keyword arguments for these ?